aggsender/flows/builder_flow_pp.go

```diff
@@ -68,6 +70,10 @@ func (p *PPBuilderFlow) GenerateBuildParams(ctx context.Context,
 	if err != nil {
 		return nil, fmt.Errorf("error applying limit size: %w", err)
```

Comment:
Make the errors consistent, WDYT?

```suggestion
		return nil, fmt.Errorf("ppFlow - error applying limit size: %w", err)
```

Reply:
Fix: https://github.com/agglayer/aggkit/pull/1264/commits/b8a346fdbddab3d94e8bda2a1790612ed4c398e4